README.md

```diff
@@ -22,6 +30,35 @@ Testing EAP or Wildfly with specific JDK version
 3. Build and run the additional testsuite activating the EAP or WILDFLY specific jdk version profile (-Deap-jdk8, -Dwildfly-jdk8).
 
 
+ADVANTAGES
+----------
+- Having all the tests at one place.
+- Comparison of the servers based on the testsuite.
+- Guarding against regression.
+
+
+MOTIVATION
+----------
+If a test is developed for server X then it can be automatically tested against all the other servers.
+
+
+FUTURE EXTENSIONS
+-----------------
+Use it with every server as a git submodule.
+
+
+PROCEDURE TO FOLLOW WHEN THE GIT SUBMODULES ARE ACTIVATED ON THE SERVERS
+------------------------------------------------------------------------
+1. Add your test at Eap Additional Testsuite and create a PR.
+2. Add your code development/fix at the server and include your eap-additional-testsuite PR as follows :
+   - cd testsuite (go to the testsuite directory of the server)
+   - git submodule update --init --remote (update your submodule)
+   - cd eap-additional-testsuite (go to the eap-additinal-testsuite directory of the server)
+   - git fetch origin refs/pull/NUM OF YOUR EAP ADDITIONAL TESTSUITE PR/head && git checkout FETCH_HEAD
+   - cd ../..
```

Comment:
Will not it be commited with the PR to the server? Should I add an instruction about git add testsuite/eap-additional-testsuite?